x-pack/packages/security-solution/features/app_features.ts

```diff
@@ -5,6 +5,6 @@
  * 2.0.
  */
 
-export { getSecurityFeature } from './src/security';
+export { getSecurityFeature, PRIVILEGE_API_TAGS } from './src/security';
```

Comment:
Hey, I love having these constants exported from the package, it's much better than hardcoding them everywhere. 
Some appreciations:

- This `app_features.ts` file is exporting only the functions needed by the _AppFeatures_ service in the Security plugin, this new export does not fit very well here. Could we create a new `privileges.ts` export?

- About naming, these api_tags are the ones for the _security_ (`siem`) feature, but we may want to do the same for the  _cases_ and _assistant_ features. Maybe we should provide a place to merge all of them within `PRIVILEGE_API_TAGS` (they are unique)? Or, otherwise, export this one under the `SECURITY_PRIVILEGE_API_TAGS` name.

- Should we expose the `PRIVILEGE_ID` from a top-level export as well? I am seeing it being imported using deep (`/src...`) paths.

Reply:
Thanks for the feedback @semd . I will make the changes you suggested above.

re: exporting

I'm ok just exporting it from the top-level. If we go that way, do you still want me to create the `privileges.ts` export namesapce?

Reply:
actually. Looks like we already have a `privileges.ts` file. I can move the exports there.

Reply:
Awesome, thanks!